aws/resource_aws_db_instance.go

```diff
@@ -775,6 +803,10 @@ func resourceAwsDbInstanceRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("monitoring_role_arn", v.MonitoringRoleArn)
 	}
 
+	if v.EnabledCloudwatchLogsExports != nil {
+		d.Set("enabled_cloudwatch_logs_exports", v.EnabledCloudwatchLogsExports)
```

Comment:
This is silently failing as `d.Set()` does not know how to handle a `[]*string` directly. You can see this from the import test:

```
--- FAIL: TestAccAWSDBInstance_importBasic (535.11s)
    testing.go:518: Step 1 error: ImportStateVerify attributes not equivalent. Difference is shown below. Top is actual, bottom is expected.
        
        (map[string]string) {
        }
        
        
        (map[string]string) (len=3) {
         (string) (len=33) "enabled_cloudwatch_logs_exports.#": (string) (len=1) "2",
         (string) (len=33) "enabled_cloudwatch_logs_exports.0": (string) (len=5) "audit",
         (string) (len=33) "enabled_cloudwatch_logs_exports.1": (string) (len=5) "error"
        }
```

A general recommendation is to always perform error checking when setting non-scalar types. This can be caught adding an import test step when working with new attributes or using `TF_SCHEMA_PANIC_ON_ERROR=1` while running the acceptance testing.

This should be something like:

```go
if err := d.Set("enabled_cloudwatch_logs_exports", flattenStringList(v.EnabledCloudwatchLogsExports)); err != nil {
  return fmt.Errorf("error setting enabled_cloudwatch_logs_exports: %s", err)
}
```

I will fix this on merge 👍